astropy/_erfa/interface.py

```diff
@@ -0,0 +1,177 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+"""
+Interfaces to the ERFA library, in particular for leap seconds.
```

Comment:
I think it's better to make this *only* about leap seconeds.  I.e., something like:

```suggestion
Interfaces to controlling leap seconds in the ERFA library.
```
and then change the name of the module file from `interface.py` to `leap_seconds.py`